app/models/form.rb

```diff
@@ -437,6 +437,8 @@ def omb_number_with_expiration_date
   def completion_rate
     if self.survey_form_activations == 0
       "N/A"
+    elsif (self.survey_form_activations <= self.response_count)
```

Comment:
let's not do this.
we can address the underlying issue instead.

Reply:
Undone